svm/tests/mock_bank.rs

```diff
@@ -138,6 +138,11 @@ pub fn program_address(program_name: &str) -> Pubkey {
     Pubkey::create_with_seed(&Pubkey::default(), program_name, &Pubkey::default()).unwrap()
 }
 
+#[allow(unused)]
+pub fn program_data_size(program_name: &str) -> usize {
+    load_program(program_name.to_string()).len()
+}
```

Comment:
what is this for?

Reply:
a very tricky edge case test in #3146 where i reallocate an account to be much larger and set a transaction data size limit that is larger than the program size plus old account size, but smaller than the program size plus new account size, to ensure that the transaction data size calculation uses the reallocated size (this affects consensus)